runtime/oti/VMHelpers.hpp

```diff
@@ -268,6 +269,10 @@ typedef enum {
 #endif /* J9VM_ARCH_AARCH64 */
 #endif /* JAVA_SPEC_VERSION >= 16 */
 
+#if JAVA_SPEC_VERSION >= 19
+typedef uintptr_t ContinuationState;
```

Comment:
This doesn't seem like the right place for this (evidence is that you're still mostly using uintptr_t in the code to represent it). Probably should move into j9nonobuilder.h and the continuation state field in the struct should use the typedef so we can consistently use it.

Reply:
We already have specific class/file ContinuationHelpers. Can't remember why, but we could not move some of more complex methods that deal with Continuation struct (like tryWinningConcurrentGCScan).

Perhaps we could move those simpler ones with deal with ContinuationState (including ContinuationState typedef) there? Scope of  VM_ContinuationHelpers:: rather than generic VM_Helpers:: would help with some of these helpers that now have rather generic name (for example, isActive)

Reply:
It seems silly to have a typedef and not using it in the struct declaration (frankly I don't really see a need for the typedef at all - the parameter name is sufficient documentation).

Reply:
I see no obvious reason why the continuation code can't be moved to the continuation-specific header file.

Reply:
In that case, I think all my concerns have been addressed.